addons/website_slides/tests/test_slide_slide.py

```diff
@@ -8,6 +8,41 @@
 from odoo.tools import mute_logger
 
 
+class TestSlidesSlideMail(slides_common.SlidesCase):
```

Comment:
Since you asked, how about a single `TestSlidesMail` class in a `test_mail.py` file sibling to the others?
You'd match "one class one file", and avoid repeating the common part of the mail template or even the tests themselves.

Reply:
Not resolved though